src/styles/blocks/stars.css

```diff
@@ -0,0 +1,29 @@
+.stars {
+  position: absolute;

```

Comment:
you still don`t need absolute here)

Reply:
fix it 
![image](https://user-images.githubusercontent.com/97547824/200163526-f6d9b7dc-9d4a-49ae-bcdb-4b15b56113aa.png)